Cargo.toml

```diff
@@ -96,14 +96,23 @@ cache_control = "=0.2.0"
 cbc = { version = "=0.1.2", features = ["alloc"] }
 # Note: Do not use the "clock" feature of chrono, as it links us to CoreFoundation on macOS.
 #       Instead use util::time::utc_now()
-chrono = { version = "0.4", default-features = false, features = ["std", "serde"] }
+chrono = { version = "0.4", default-features = false, features = [
+  "std",
+  "serde",
+] }
 console_static_text = "=0.8.1"
 data-encoding = "2.3.3"
 data-url = "=0.3.0"
 deno_cache_dir = "=0.7.1"
 dlopen2 = "0.6.1"
 ecb = "=0.1.2"
-elliptic-curve = { version = "0.13.4", features = ["alloc", "arithmetic", "ecdh", "std", "pem"] }
+elliptic-curve = { version = "0.13.4", features = [
+  "alloc",
+  "arithmetic",
+  "ecdh",
+  "std",
+  "pem",
+] }
```

Comment:
This kind of formatting takes up a lot of vertical space for not very useful information.